TheBTeam.ConsoleApp/MainMenu.cs

```diff
@@ -123,6 +126,30 @@ public static void ShowMainMenu()
                     ConsoleFactory.EditTransaction(tmpListTransactions.TransactionsList[indexOfTransaction]);
                     Console.ReadKey();
                 }
+                else if (mainMenuItem[currentItem] == ("Search transactions by date"))
```

Comment:
Logic HAVE TO bE in BLL not console

Reply:
@PuchalskiMateusz  All menu is build around this logic (please take a look at whole MainMenu.cs